api/app/tests/apps/sigmax/test_send_example.py

```diff
@@ -0,0 +1,26 @@
+import os
+from unittest import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+REQUIRED_ENV = {'SIGMAX_AUTH_TOKEN': 'TEST', 'SIGMAX_SERVER': 'https://example.com'}
+
+
+class CommandTestCase(TestCase):
+    @mock.patch('requests.post', autospec=True)
+    @mock.patch.dict(os.environ, REQUIRED_ENV)
```

Comment:
Can't you use the Django util `override_settings` here as well?

Reply:
the whole send_example flow is obsolete, and will be removed